Source/Animation.swift

```diff
@@ -83,8 +90,13 @@ extension Animation {
     }
 
     @discardableResult
-    public func animate(in view: UIView) -> AnimationCanceller {
+    public func animate(in view: View) -> AnimationCanceller {
+        #if os(iOS) || os(watchOS) || os(tvOS)
         return animate(in: view.layer)
+        #elseif os(OSX)
+        view.wantsLayer = true
+        return animate(in: view.layer!)
```

Comment:
Do not use force unwrap.
I think this code is better.

```swift
if let layer = view.layer {
    return animate(in: layer)
} else {
    fatalError("view.layer is nil in \(#file)_\(#function)_\(#line)")
}
```

Reply:
I agree, but `view.wantsLayer = true` ensures that the layer exists, then the check IMHO is innecesary and in case de error it will be super explicit with force unwrap. What you thin?

Reply:
`view.wantsLayer = true` maybe ensures that the layer exists, but it is not ensure as a non-optional type.
If it can not be represent as non-optional type, I think it should clearly describe error case for that.